CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 ## [Unreleased]
 ## [1.1.0]
 ### Added
+- Add support for making multiple requests, with a threshold for success
```

Comment:
This is not adding support for multiple requests (which I would classify as retries) This is changing to a % based approach assuming you will send multiple requests and alert based on that. Please move this under the UNRELEASED